src/lib_c/x86_64-windows-msvc/c/winnt.cr

```diff
@@ -8,6 +8,7 @@ lib LibC
   alias LPWSTR = WCHAR*
   alias LPWCH = WCHAR*
 
+  alias LPVOID = Void*
```

Comment:
Ditto, `Void*` instead of `LPVOID*` is fine.

Reply:
Removed this.